reeds/function_libs/pipeline/worker_scripts/simulation_workers/RE_EDS_simulation_run_worker.py

```diff
@@ -74,6 +74,16 @@ def work(out_dir: str, in_coord: str, in_imd_path: str, in_topo_path: str, in_pe
         # get file outprefix
         out_prefix = os.path.splitext(os.path.basename(imd_path))[0]
 
+        #TODO: This is a stupid workaround as Euler tends to place nans in the euler angles, that should not be there!
+        from pygromos.files.coord import cnf
+        import math
+        import glob
+        for in_cnf_tmp in glob.glob(os.path.dirname(in_coord)+"/*.cnf"):
+            cnf_file = cnf.Cnf(in_cnf_tmp)
+            if(hasattr(cnf_file, "GENBOX") and any([math.isnan(x) for x in cnf_file.GENBOX.euler])):
```

Comment:
nice!

Reply:
This looks much better than the sed command I had as a temporary fix (around line 241). 
`reeds/reeds/function_libs/pipeline/jobScheduling_scripts/scheduler_functions.py`
 
We can probably remove these sed commands now. As well as the previous grep commands which were there to check fr nans.

Reply:
Hm I would keep your changes for the moment, as they might be required by the analysis script :)